app/src/main/java/org/oppia/app/databinding/EditTextBindingAdapters.java

```diff
@@ -0,0 +1,20 @@
+package org.oppia.app.databinding;
+
+import android.text.TextWatcher;
+import android.widget.EditText;
+
```

Comment:
As mentioned above, no newlines between imports for Google Java style except when using static imports. See: https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing.

Reply:
Fixed!